sdks/python/container/py311/base_image_requirements.txt

```diff
@@ -73,7 +73,6 @@ google-cloud-vision==3.7.2
 google-crc32c==1.5.0
 google-resumable-media==2.7.0
 googleapis-common-protos==1.63.0
-greenlet==3.0.3
```

Comment:
why is greenlet removed from this file?
https://github.com/apache/beam/actions/runs/8694084666/job/23842253554?pr=30962

I just ran `./gradlew :sdks:python:container:generatePythonRequirementsAll` on my mac.

@tvalentyn Do you know?

Reply:
most likely because you ran on the mac. Revert that line manually. Thanks!

Reply:
Got it. Thanks.